src/App.tsx

```diff
@@ -1,56 +1,122 @@
-import React from 'react';
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+import React, { useState, useMemo } from 'react';
 import './App.scss';
+import debounce from 'lodash.debounce';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [query, setQuery] = useState('');
+  const [selectedWord, setSelectedWord] = useState('');
+  const [selectedPerson, setSelectedPerson] = useState(peopleFromServer[0]);
+  const [isDebouncing, setIsDebouncing] = useState(false);
+
+  let isEntered = false;
+  let currentPerson = '';
+
+  if (query.length > 0) {
+    isEntered = true;
+  } else {
+    isEntered = false;
+  }
+
+  const foundPerson
+   = peopleFromServer.find(people => people.name === selectedPerson.name);
+
+  if (foundPerson !== undefined) {
+    currentPerson = foundPerson?.name;
+  }
+
+  const visibleMovies: Person[] = useMemo(
```

Comment:
Are you sure `visibleMovies` is an appropriate name for this variable? Seems that visiblePeople will fit better here